assets/airports/kjfk.json

```diff
@@ -7,227 +8,565 @@
-                "GREKI": ["GREKI"],
-                "MERIT": ["MERIT"],
+                "ARD": ["ARD"],
                 "BAYYS": ["BAYYS"],
                 "BDR": ["BDR"],
                 "BETTE": ["BETTE"],
-                "HAPIE": ["HAPIE"],
-                "SHIPP": ["SHIPP"],
-                "WAVEY": ["WAVEY"],
-                "WHITE": ["WHITE"],
+                "COATE": ["COATE"],
                 "DIXIE": ["DIXIE"],
-                "RNGRR": ["RNGRR"],
+                "GAYEL": ["GAYEL"],
+                "GREKI": ["GREKI"],
+                "HAAYS": ["HAAYS"],
+                "HAPIE": ["HAPIE"],
+                "LANNA": ["LANNA"],
+                "MERIT": ["MERIT"],
+                "NEION": ["NEION"],
                 "RBV": ["RBV"],
-                "ARD": ["ARD"],
+                "RNGRR": ["RNGRR"],
+                "SAX": ["SAX"],
                 "SBJ": ["SBJ"],
-                "LANNA": ["LANNA"]
+                "SHIPP": ["SHIPP"],
+                "WAVEY": ["WAVEY"],
+                "WHITE": ["WHITE"]
+            },
+            "draw": [
+                ["COATE*"],
+                ["NEION*"],
+                ["HAAYS*"],
+                ["GAYEL*"],
+                ["DEEZZ*"],
+                ["GREKI*"],
+                ["MERIT*"],
+                ["BAYYS*"],
+                ["BETTE*"],
+                ["HAPIE*"],
+                ["SHIPP*"],
+                ["WAVEY*"],
+                ["WHITE*"]
+            ]
+        },
+        "BETTE6": {
+            "icao": "BETTE6",
+            "name": "Betty Six",
+            "rwy": {
+                "KJFK4L" : ["#099"],
+                "KJFK4R" : ["#099"],
+                "KJFK13L": ["#134"],
+                "KJFK13R": ["#134"],
+                "KJFK22L": ["#224"],
+                "KJFK22R": ["#224"],
+                "KJFK31L": ["CRI", ["_CRI176002", "A25+"], "#176"],
+                "KJFK31R": ["CRI", ["_CRI176002", "A25+"], "#176"]
+            },
+            "body": [],
+            "exitPoints": {
+                "BETTE": ["BETTE"]
+            },
+            "draw": [
+                ["BETTE*"]
+            ]
+        },
+        "DEEZZ5": {
+            "icao": "DEEZZ5",
+            "name": "Deez Five",
+            "rwy": {
+                "KJFK4L" : ["_RWY22R01DME", "#099"],
+                "KJFK4R" : ["_RWY22L01DME", "#099"],
+                "KJFK13L": ["_RWY31R01DME", "#109"],
+                "KJFK13R": ["_RWY31L01DME", "#109"],
+                "KJFK22L": ["#224"],
+                "KJFK22R": ["#224"],
+                "KJFK31L": ["_RWY13RSKORR057", "SKORR", ["CESID", "A25+"], "YNKEE", "#172"],
+                "KJFK31R": ["_RWY13LSKORR062", "SKORR", ["CESID", "A25+"], "YNKEE", "#172"]
+            },
+            "body": ["DEEZZ", "HEERO"],
+            "exitPoints": {
+                "CANDR": ["CANDR"],
+                "HEERO": ["HEERO"],
+                "TOWIN": ["TOWIN"]
+            },
+            "draw": [
```

Comment:
Lines 355 to 357 should reflect comments made on lines 347 to 349

Reply:
done